backend/src/zimfarm_backend/routes/auth/logic.py

```diff
@@ -49,7 +50,7 @@ def _access_token_response(db_session: OrmSession, db_user: User, response: Resp
             user_id=str(db_user.id),
             issue_time=issue_time,
             username=db_user.username,
-            scope=db_user.scope,
+            scope=ROLES.get(db_user.role, db_user.scope),
```

Comment:
Looks like you've spotted a bug which was not catch in any test (I'm not sure since you've force-pushed). If you confirm, please add a test to ensure that scope in access token is properly populated even if user only has a role.